TRAVIS

```diff
@@ -1,6 +1,13 @@
 Implement continuous integration
 
-Implement a track test suite that can run both locally and on Travis CI. The track test suite should verify that each exercise makes sense, by running the exercise tests against the example solution.
+Implement the "track test suite".  This suite should:
+
+- for each exercise in the track,
+  - run that exercise's test suite (aka the "exercise test suite");
+  - use the exercise's solution as the code under test.
+- ensure all test cases are enabled;
+- be executed on Travis CI;
+- be easy to run on a contributor/maintainer's local machine (i.e. a build script with any needed instructions).
 
```

Comment:
Ah, yes. Instructions. Let's specify that this needs to be documented in the README of the track.